tracer/src/Datadog.Trace.Annotations/TraceAttribute.cs

```diff
@@ -0,0 +1,29 @@
+// <copyright file="TraceAttribute.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
```

Comment:
nit: Could enable this at the project level instead 🙂

Reply:
Implemented in 46ae5f8dd067fd13db2b590f9a283f7ed6c1edea

Reply:
Done in 46ae5f8dd067fd13db2b590f9a283f7ed6c1edea